### PR TITLE
control whether arguments should be inlined or passed in.

### PR DESCRIPTION
**Pull request by lure**
Provides a means to control whether arguments should be inlined in a final query.

### CODE REVIEW

**Comment by Tapac** [comment]
Can you please highlight a use-case?

**Comment by bog-walk** [comment]
@lure Thanks for submitting all these PRs for your feature requests.

As we get the ball rolling on reviewing them, I'd just like to confirm 2 things:

1. In the efforts of providing better documentation moving forward, I'd like to properly understand the advantage that the proposed change provides to query statements. e.g. I've come across users that leverage their own versions with `prepared = false` to enable raw SQL generation without final query execution. Is this the purpose of the enhancement or is there an additional use-case that benefits from this?
2. Quite a bit has changed since this particular PR was submitted. Would you be willing to push an updated commit resolving the conflicts or would you rather we perform the necessary edits?

**Comment by lure** [comment]
@bog-walk  
re 1: yes, this is to be able to benefit from automatic sql generation required to build more complicated DML / DDL prepositions from scratches. 
re 2: will do asap

**Comment by lure** [comment]
@bog-walk 
@joc-a 
@Tapac 

updated, fixed. For some reason the validation has not passed and I do not know how to restart it

File: exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Column.kt [hunk 1]
```diff
@@ -35,6 +35,8 @@ class Column<T>(
     var defaultValueFun: (() -> T)? = null
     internal var dbDefaultValue: Expression<T>? = null
 
+    fun defaultValueInDb() = dbDefaultValue
```
Comment (bog-walk) [inline exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Column.kt:38]: Is this edit just to bypass the visibility modifier?

**Review by bog-walk — APPROVED** [review]

**Review by e5l — APPROVED** [review]

**Pull request merged**